src/main/resources/templates/gardens/list.html

```diff
@@ -79,8 +77,19 @@ <h5 class="card-title" th:text="${garden.name}" ></h5>
                                 <p class="card-text" th:text="${'Owner: ' + garden.owner.name}"></p>
                                 <p class="card-text" style="margin-bottom: 0"><strong>About our garden...</strong></p>
                                 <p class="card-text show-read-more" th:text="${garden.description}"></p>
-                                <a class="card-text btn btn-primary" th:href="@{'/user/view/' + ${garden.id}}" style="margin-bottom:10px;">More Information</a>
-                                <a class="card-text btn btn-success" id="add" th:href="@{'/user/gardens/tour/' + ${garden.id}}">Add to Tour</a>
+                                <div style="text-align:center;">
+                                    <a class="card-text btn-sm btn-primary" th:href="@{'/user/view/' + ${garden.id}}">More Information</a>
+                                    <br>
+                                    <div style="margin-top:5px">
+                                        <a class="card-text btn-sm btn-success" id="add" th:href="@{'/user/gardens/tour/' + ${garden.id}}" style="margin-right:10px;">Add to Tour</a>
+                                        <form method="GET" th:action="@{/user/map}" style="display:inline;">
+                                            <button class="card-text btn-sm btn-info"
```

Comment:
i randomly picked btn-info for the individual map view, and I saw @jamiander that you picked that color for the return to water garden tour button, i'll pick another color. We're running out of standard bootstrap button colors! :)

Reply:
No problem, whatever works.  They need more colors!